lib/llvm-backend/src/platform/win.rs

```diff
@@ -0,0 +1,80 @@
+use super::common::round_up_to_page_size;
+use crate::structs::{LLVMResult, MemProtect};
+use std::ptr;
+
+use winapi::um::memoryapi::{VirtualAlloc, VirtualFree};
+use winapi::um::winnt::{
+    MEM_COMMIT, MEM_DECOMMIT, MEM_RESERVE, PAGE_EXECUTE_READ, PAGE_NOACCESS, PAGE_READONLY,
+    PAGE_READWRITE,
+};
+
+pub unsafe fn visit_fde(_addr: *mut u8, _size: usize, _visitor: extern "C" fn(*mut u8)) {
+    // Do nothing on Windows
```

Comment:
Question, is this ever called on Windows?

Reply:
Yup, from the `object_loader.cpp`